pages/git-team.tsx

```diff
@@ -0,0 +1,41 @@
+import { GetStaticProps, NextPage } from 'next';
+import { join } from 'path';
+import BackLink from '../components/BackLink';
+import Layout from '../components/Layout/Layout';
+import PageContainer from '../components/Layout/PageContainer';
+import HtmlDisplay from '../components/HtmlDisplay';
+import {
+  getMarkdownContent,
+  markdownDir,
+  MarkdownProps,
+} from '../lib/markdown';
+import Heading1 from '../components/Heading1';
+
+export const getStaticProps: GetStaticProps = async () => {
+  const markdownContent = await getMarkdownContent(
+    join(markdownDir, 'articles', 'git-team.md')
+  );
+  return {
+    props: {
+      ...markdownContent,
+    },
+  };
+};
+
+const GitTeam: NextPage<MarkdownProps> = ({
+  contentHtml,
+  title,
+  description,
+}) => {
+  return (
+    <Layout title={title} description={description || ''}>
```

Comment:
Warto by było wyciągnąć to wszystko do osobnego komponentu i przekazać tylko to co potrzebne, żeby nie musieć tyle kodu dodawać przy każdym artykule.

Reply:
Próbowałem to ogarnąć z pomocą Microsoft Copilota, ale poległem. Nie ogarniam tego Next.js jak tam te magie działają :(

Reply:
Okej, to zostaw jak jest. Teraz już mi świta, że część z tego to Nextowy boilerplate.